lib/browser.rb

```diff
@@ -1,15 +1,22 @@
 class Browser
   def initialize(options = {})
     @found_jobs = []
-    @keywords = options[:keywords] || "ruby, rails"
+    @keywords = options[:keywords] || "product"
     @link_parser = LinkParser.new
     @url_validator = UrlValidator.new
+
+    @num_pages = ARGV[0] ? ARGV[0].to_i : 1
   end
 
   def scour_aggregator(aggregator)
     @aggregator = aggregator
-    page = browse_to_url(@aggregator.url)
-    scour_page(page)
+
+    (1..@num_pages).each do |page_id|
+      @aggregator.url = "https://nytm.org/made?list=true&page=#{page_id}"
```

Comment:
The test for this section is now failing because it tries to set the URL manually with test data. I'd like to move this out of the method to get the test to pass, let me know how you'd prefer I do that.